store/getters.ts

```diff
@@ -20,6 +20,7 @@ export const getters: GetterTree<PaymentServiceState, any> = {
     return state.paymentStatusFetched
   },
   getPspMethods (state) {
```

Comment:
@cowoder FYI: this is method should return the Array type. (Bug in return the Object type.)